oxidation/libparsec/crates/types/src/organization.rs

```diff
@@ -21,6 +21,7 @@ pub struct OrganizationConfig {
     pub sequester_authority: Option<Vec<u8>>,
     // TODO: Should this be `SequesterServiceCertificate` instead of bytes?
     pub sequester_services: Option<Vec<Vec<u8>>>,
+    pub minimum_archiving_period: u64,
```

Comment:
Use [`libparsec_types::time::Duration`](https://docs.rs/chrono/latest/chrono/struct.Duration.html) instead

The is no Python binding to `Duration` though, but I think it's okay to use `int`  in Python.
The conversion to `i64` [is pretty trivial btw](https://docs.rs/chrono/latest/chrono/struct.Duration.html#method.num_seconds)

Reply:
Do we really want this? 

> This also allows for the negative duration;

Reply:
I was wondering, but negative duration doesn't seem like a big issue given we only do `now - archiving_start > minimum_archiving_period`

The good point about using a `Duration` type is we are much more explicit about what this type represent, with a `u64` we always wonder if it's day ? seconds ? milliseconds ? etc.

The other solution is to add the unit in the name of the variable, but I find it is just better to use `Duration` instead (unless in the Python bindings were we definitely don't want to bother with more binding work for this ^^).

Reply:
[fb3447e](https://github.com/Scille/parsec-cloud/pull/5237/commits/fb3447e5d81652455a04a0649162772c97abe03e)